### ISSUE TITLE
Cannot access dropdown menus via Tab button (a11y violation)

### ISSUE THREAD

**Issue by quinnouyang** [quinnouyang/contextual-engineering-site#60]
<img width="509" alt="image" src="https://user-images.githubusercontent.com/90884224/210275656-2058f3c8-31a7-4716-bc9f-70312f341c90.png">
